keda/Chart.yaml

```diff
@@ -8,11 +8,11 @@ kubeVersion: ">=v1.23.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.11.2
+version: 2.11.3
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 2.11.2
+appVersion: 2.11.3
```

Comment:
Could you revert these changes?

Reply:
Just curious, so we are overriding the existing helm chart version instead of bumping it up right?

Reply:
If we make this a breaking change, will we still need to bump the minor chart version?

Reply:
my bad. Wrong line sorry :facepalm: